server/src/main/java/io/crate/auth/HostBasedAuthentication.java

```diff
@@ -96,11 +96,17 @@ static SSL parseValue(String value) {
     private final Roles roles;
     private final DnsResolver dnsResolver;
 
+    private final Supplier<String> clusterId;
+
     @Inject
```

Comment:
Not related to this PR, but I think we instantiate it ourselves nowadays so the guide annotations could be removed?

Reply:
removed